Converters/CSi/Speckle.Converters.CSiShared/ToSpeckle/Helpers/CsiJointPropertiesExtractor.cs

```diff
@@ -0,0 +1,67 @@
+using Speckle.Converters.Common;
+using Speckle.Converters.CSiShared.Extensions;
+using Speckle.Converters.CSiShared.Utils;
+
+namespace Speckle.Converters.CSiShared.ToSpeckle.Helpers;
+
+/// <summary>
+/// Extracts properties common to joint elements across CSi products (e.g., ETABS, SAP2000)
+/// using the PointObj API calls.
+/// </summary>
+/// <remarks>
+/// Design Decisions:
+/// - Individual methods preferred over batched calls due to:
+///   * Independent API calls with no performance gain from batching (?)
+///   * Easier debugging and error tracing
+///   * Simpler maintenance as each method maps to one API concept
+/// Responsibilities:
+/// - Provides a focused interface for extracting properties specific to joint elements.
+/// - Ensures consistency in property extraction logic across supported CSi products.
+/// Integration:
+/// - Part of the property extraction hierarchy
+/// - Used by <see cref="SharedPropertiesExtractor"/> for delegating joint property extraction
+/// </remarks>
+public sealed class CsiJointPropertiesExtractor
+{
+  private readonly IConverterSettingsStore<CsiConversionSettings> _settingsStore;
+
+  public CsiJointPropertiesExtractor(IConverterSettingsStore<CsiConversionSettings> settingsStore)
+  {
+    _settingsStore = settingsStore;
+  }
+
+  public void ExtractProperties(CsiJointWrapper joint, PropertyExtractionResult jointData)
+  {
+    jointData.ApplicationId = GetApplicationId(joint);
```

Comment:
This could be replaced with `joint.GetSpeckleApplicationId()` and the function on line 42 removed. Same for other extractors!

Reply:
True! Done (commit: applicationId simplification)